tests/device_tests/cardano/test_sign_tx.py

```diff
@@ -100,48 +109,48 @@ def test_cardano_sign_tx_failed(client, parameters, result):
             )
 
 
-@parametrize_using_common_fixtures("cardano/sign_tx.chunked.json")
-def test_cardano_sign_tx_with_multiple_chunks(client, parameters, result):
-    inputs = [cardano.parse_input(i) for i in parameters["inputs"]]
-    outputs = [cardano.parse_output(o) for o in parameters["outputs"]]
-    certificates = [cardano.parse_certificate(c) for c in parameters["certificates"]]
-    withdrawals = [cardano.parse_withdrawal(w) for w in parameters["withdrawals"]]
-    auxiliary_data = cardano.parse_auxiliary_data(parameters["auxiliary_data"])
-
-    input_flow = parameters.get("input_flow", ())
-
-    expected_responses = [
-        messages.PassphraseRequest(),
-        messages.ButtonRequest(),
-        messages.ButtonRequest(),
-    ]
-    expected_responses += [
-        messages.CardanoSignedTxChunk(signed_tx_chunk=bytes.fromhex(signed_tx_chunk))
-        for signed_tx_chunk in result["signed_tx_chunks"]
-    ]
-    expected_responses += [
-        messages.CardanoSignedTx(tx_hash=bytes.fromhex(result["tx_hash"]))
-    ]
-
-    with client:
-        client.set_input_flow(_to_device_actions(client, input_flow))
-        client.set_expected_responses(expected_responses)
-
-        response = cardano.sign_tx(
-            client=client,
-            inputs=inputs,
-            outputs=outputs,
-            fee=parameters["fee"],
-            ttl=parameters.get("ttl"),
-            validity_interval_start=parameters.get("validity_interval_start"),
-            certificates=certificates,
-            withdrawals=withdrawals,
-            protocol_magic=parameters["protocol_magic"],
-            network_id=parameters["network_id"],
-            auxiliary_data=auxiliary_data,
-        )
-        assert response.tx_hash.hex() == result["tx_hash"]
-        assert response.serialized_tx.hex() == result["serialized_tx"]
+# @parametrize_using_common_fixtures("cardano/sign_tx.chunked.json")
+# def test_cardano_sign_tx_with_multiple_chunks(client, parameters, result):
```

Comment:
This test as it is doesn't make much sense now, but later all the tests will have to check the expected responses. Keeping it until I see how the tests will look like with streamed tx data.

Reply:
I'm not sure all the tests will need to check the expected responses, just as they aren't doing it now. I think it's going to be enough to have just one test which checks all the messages (like this test) and others will just check the results. But yeah, we'll see.